airbyte-webapp/src/components/connectorBuilder/StreamTestingPanel/StreamSelector.module.scss

```diff
@@ -11,14 +11,19 @@
 .button {
   padding: variables.$spacing-md;
   background-color: transparent;
-  border-radius: variables.$border-radius-lg;
+  border-radius: variables.$border-radius-md;
   border: none;
   display: flex;
```

Comment:
In this whole file: Instead of using margins to position stuff, you can use flex layout:
on the button container, set
```
justify-content: center;
gap: variables.$spacing-md;
```

This will put both elements in the center of the flex container with the specified gap. Then you don't need any of the other margins.